### DISCUSSION TITLE
技术分享: 以太坊scaling

### DISCUSSION THREAD

**Discussion by shanjiefengyi** [rebase-network/hello-world#63] [Event]
随着区块链生态的不断发展，链上交易也越来越多，同时造成了巨大的链上拥堵，因此扩容方案的推出非常重要，本周六Meter CEO祝小翰将分享Meter在以太坊扩容中的实践。

分享时间：周六下午1点（3月12日）
分享主题：以太坊scaling
分享地点：rebase技术社区
分享人：Meter CEO

**YouTube视频链接：**
https://youtu.be/mKjY4TYGv48

**分享资料(PPT)：**
[区块链共识和扩容.pdf](https://github.com/rebase-network/hello-world/files/8245748/default.pdf)

**Comment by luckyyang** [comment]
会议主题：Rebase大学技术分享: 以太坊scaling
会议时间：2022/03/12 13:00-14:00 (GMT+08:00) 中国标准时间 - 北京

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/TdpXDycPTtKk

#腾讯会议：629-310-795

**Comment by luckyyang** [comment]
Meter 黑客松信息: https://github.com/rebase-network/event-pool/issues/2

**Comment by luckyyang** [comment]
合成资产链接 https://www.sumer.money/
